src/Searchlight/DataSource.cs

```diff
@@ -33,6 +34,7 @@ public class DataSource
         /// The field name of the default sort field, if none are specified.
         /// This is necessary to ensure reliable pagination.
         /// </summary>
+        [Required(ErrorMessage = "DefaultSort is required")]
```

Comment:
I need some sort of test that shows that this works as expected.  For example, you need to demonstrate that adding a table without a default sort value fails or throws an error that explains why the table can't be added to the searchlight engine.

Reply:
I don't think the required attribute works, so I ended up creating a new exception and throwing it in the DataSource Create method if DefaultSort is null or is not a valid property.